proto/arkeo/arkeo/genesis.proto

```diff
@@ -13,5 +13,5 @@ message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
   repeated Provider providers = 2 [ (gogoproto.nullable) = false ];
   repeated Contract contracts = 3 [ (gogoproto.nullable) = false ];
-  // this line is used by starport scaffolding # genesis/proto/state
+  uint64 next_contract_id = 4;
```

Comment:
ignite is actually need this line of comment to work , if we remove , it will complain about it.

Reply:
will add back.